docs/source/development_release.rst

```diff
@@ -8,11 +8,13 @@ notebook.
 
 Check installed tools
 ---------------------
-Review :ref:`development_js`. Make sure all the tools needed to generate the
+
+Review ``CONTRIBUTING.rst``. Make sure all the tools needed to generate the
```

Comment:
I don't know how, that's why I showed only the file name.

The Sphinx-way to do that would probably be the `:download:` role, but that links to a *copy* of the file, which is not the file where potential changes are supposed to be made.